providers/curation/github.js

```diff
@@ -379,14 +379,34 @@ ${this._formatDefinitions(patch.patches)}`
   }
 
   /**
-   * Given partial coordinates, return a list of full coordinates for which we have merged curations
+   * Given partial coordinates, return a list of Curations and Contributions
    * @param {EntityCoordinates} coordinates - the partial coordinates that describe the sort of curation to look for.
    * @returns {[EntityCoordinates]} - Array of coordinates describing the available curations
    */
   async list(coordinates) {
     return this.store.list(coordinates)
   }
 
+  /**
+   * Return a list of Curations and Contributions for each coordinates provided
+   *
+   * @param {*} coordinatesList - an array of coordinate paths to list
+   * @returns A list of Curations and Contributions for each coordinates provided
```

Comment:
A bit confused as to what the result looks like here. 

Also, what is a `Contribution`?

Reply:
A curation is the change that is overlayed on the definition, a contribution is the actual PR

Here is a sample

request:
```
POST /curations
["nuget/nuget/-/AsyncIO/0.1.25","nuget/nuget/-/Newtonsoft.Json/11.0.2"]
```

response:
```json
{
  "nuget/nuget/-/AsyncIO/0.1.25": {
    "curations": {},
    "contributions": [
      {
        "pr": {
          "number": 401,
          "id": 255122478,
          "state": "open",
          "title": "Test changing facets",
          "body": "\n**Type:** Other\n\n**Summary:**\nTest changing facets\n\n**Details:**\nTest changing facets\n\n**Resolution:**\nTest changing facets\n\n**Affected definitions**:\n- AsyncIO 0.1.25",
          "created_at": "2019-02-21T17:51:00Z",
          "updated_at": "2019-02-21T17:51:00Z",
          "closed_at": null,
          "merged_at": null,
          "merge_commit_sha": "77bb9c4701008bb558845322b83f7b9ea072d6cd",
          "user": { "login": "storrisi" },
          "head": { "sha": "494a3f902235d7d8957b248c1e9fecd0c181da34", "repo": { "id": 115941547 } },
          "base": { "sha": "1f8ee8bbe8200c494bdfa458b5b589dc5c0d9862", "repo": { "id": 115941547 } }
        },
        "files": [
          {
            "path": "curations/nuget/nuget/-/AsyncIO.yaml",
            "coordinates": { "type": "nuget", "provider": "nuget", "name": "asyncio" },
            "revisions": [{ "revision": "0.1.25", "data": { "described": { "facets": { "data": ["**/*.nuspec"] } } } }]
          }
        ]
      },
      {
        "pr": {
          "number": 361,
          "id": 249659624,
          "state": "closed",
          "title": "clearlydefinedbot_190201_162514.524",
          "body": "\n**Type:** Missing\n\n**Summary:**\nAUTOMATION TEST\n\n**Details:**\nAUTOMATION TEST\n\n**Resolution:**\nAUTOMATION TEST\n\n**Affected definitions**:\n- AsyncIO 0.1.25",
          "created_at": "2019-02-01T16:25:16Z",
          "updated_at": "2019-02-15T21:13:59Z",
          "closed_at": "2019-02-15T21:13:59Z",
          "merged_at": null,
          "merge_commit_sha": "0702ea460b4626bddcf474a9b1f4752a45a883cb",
          "user": { "login": "clearlydefinedbot" },
          "head": { "sha": "b69a6faea44f8300258d1b2fd3039e78dc976ebb", "repo": { "id": 115941547 } },
          "base": { "sha": "d62e5a6ca6b7d3a491c3c8286dbe95a1b8a36111", "repo": { "id": 115941547 } }
        },
        "files": [
          {
            "path": "curations/nuget/nuget/-/AsyncIO.yaml",
            "coordinates": { "type": "nuget", "provider": "nuget", "name": "asyncio" },
            "revisions": [{ "revision": "0.1.25", "data": { "licensed": { "declared": "MIT-0" } } }]
          }
        ]
      }
    ]
  },
  "nuget/nuget/-/Newtonsoft.Json/11.0.2": {
    "curations": { "nuget/nuget/-/Newtonsoft.Json/11.0.2": { "licensed": { "declared": "MIT" } } },
    "contributions": [
      {
        "pr": {
          "number": 187,
          "id": 211552575,
          "state": "closed",
          "title": "clearlydefinedbot_180828_204556.030",
          "body": "set Newtonsoft.Json Declared license to 'MIT'",
          "created_at": "2018-08-28T20:45:57Z",
          "updated_at": "2018-08-28T20:47:29Z",
          "closed_at": "2018-08-28T20:47:26Z",
          "merged_at": "2018-08-28T20:47:26Z",
          "merge_commit_sha": "09c4cda6b89742f9a59707feebc5fa9fc97fa9af",
          "user": { "login": "clearlydefinedbot" },
          "head": { "sha": "7a2c230c25c51a00d251cba3056f28ad88d3a380", "repo": { "id": 115941547 } },
          "base": { "sha": "f91ddb1c6f0fcd46b33befb6cf94f5e141f9be98", "repo": { "id": 115941547 } }
        },
        "files": [
          {
            "path": "curations/nuget/nuget/-/Newtonsoft.Json.yaml",
            "coordinates": { "type": "nuget", "provider": "nuget", "name": "newtonsoft.json" },
            "revisions": [{ "revision": "11.0.2", "data": { "licensed": { "declared": "MIT" } } }]
          }
        ]
      }
    ]
  }
}

```